### PR TITLE
DIGI_7_ListAllAvaliableMenuItems

### PR DESCRIPTION
**Pull request by praveen957**
1.MenuItemDataModel is added for to get data
2.MenuItemController is modified for get model

### CODE REVIEW

File: DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs [hunk 1]
```diff
@@ -1,13 +1,39 @@
...
 {
     public class MenuItemController : ApiController
     {
+        MenuItemDataModel menuItemDataModel = null;
+        DataTable dtRecords = null;
+
+        //Author: praveen kumar
+        //6-25-2020
+        //Description: To get all avaialbel MenuItems from database
+        //Reviewer
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs:19]: Add revier name as Bhargavkrishna Ane

File: DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs [hunk 2]
```diff
@@ -1,13 +1,39 @@
...
+        {
+            try
+            {
+                menuItemDataModel = new MenuItemDataModel();
+                dtRecords = menuItemDataModel.getAllMenuItems();
+                dtRecords.TableName = "MenuItems";
+            }
+            catch (Exception ex)
+            {
+                return Request.CreateErrorResponse(HttpStatusCode.NotFound, "Record Not Found");
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs:34]: Change status code to  internalserver error and message as "Unable to process request"

File: DigiOrderApi/DigiOrderApiDataAccess/MenuItemDataModel.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+{
+    public class MenuItemDataModel
+    {
+        DataTable dtResult = null;
+
+        //Author: praveen kumar
+        //6-25-2020
+        //Description: To get all avaialbel MenuItems from database
+        //Reviewer
+        //Review Version:
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApiDataAccess/MenuItemDataModel.cs:15]: Remove comments

File: DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs (hunk 1 above)
Comment (rajendarreddy1234) [reply DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs]: Resolved

File: DigiOrderApi/DigiOrderApiDataAccess/MenuItemDataModel.cs (hunk 3 above)
Comment (rajendarreddy1234) [reply DigiOrderApi/DigiOrderApiDataAccess/MenuItemDataModel.cs]: Resolved

File: DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs (hunk 2 above)
Comment (rajendarreddy1234) [reply DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs]: Resolved

File: DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs [hunk 4]
```diff
@@ -1,13 +1,39 @@
...
+                menuItemDataModel = new MenuItemDataModel();
+                dtRecords = menuItemDataModel.getAllMenuItems();
+                dtRecords.TableName = "MenuItems";
+            }
+            catch (Exception ex)
+            {
+                return Request.CreateErrorResponse(HttpStatusCode.InternalServerError, "Unable to process request");
+            }
+            return Request.CreateResponse(HttpStatusCode.OK, dtRecords);
+        }
```
Comment (anebhargavkrishna) [inline DigiOrderApi/DigiOrderApi/Controllers/MenuItemController.cs:37]: add digiorderapi.csproj file

**Pull request merged**